event/subscriber.go

```diff
@@ -0,0 +1,77 @@
+package event
+
+import (
+	"context"
+	"fmt"
+	"github.com/medibloc/panacea-doracle/config"
+	log "github.com/sirupsen/logrus"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+	"time"
+)
+
+type PanaceaSubscriber struct {
+	Client *rpchttp.HTTP
+}
+
+// NewSubscriber generates a rpc http client with websocket address.
+func NewSubscriber(conf *config.Config) (*PanaceaSubscriber, error) {
+	client, err := rpchttp.New(conf.Panacea.WSAddr, "/websocket")
+	if err != nil {
+		return nil, err
+	}
+
+	err = client.Start()
+	if err != nil {
+		return nil, err
+	}
+
+	return &PanaceaSubscriber{
+		Client: client,
+	}, nil
+}
+
+func (s *PanaceaSubscriber) Run(events ...Event) error {
+	log.Infof("start panacea event subscriber")
+
+	for _, e := range events {
+		query := e.GetEventType() + "." + e.GetEventAttributeKey() + "=" + e.GetEventAttributeValue()
+
+		ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+		defer cancel()
+
+		fmt.Println(query)
+
+		txs, err := s.Client.Subscribe(ctx, "", query)
+		if err != nil {
+			return err
+		}
+
+		e := e
```

Comment:
```suggestion
```

Reply:
If I deleted this, there has a lint issue. So I'll check and fix it and ping re-review to you.

Reply:
Fixed in [37ab159](https://github.com/medibloc/panacea-doracle/pull/15/commits/37ab159f1a1202f0347b14dedb4374f52e42c5fa).